packages/app-extensions/src/appFactory/App.js

```diff
@@ -2,19 +2,15 @@ import PropTypes from 'prop-types'
 import React, {useCallback} from 'react'
 import {IntlProvider} from 'react-intl-redux/lib'
 import {Provider} from 'react-redux'
-import styled from 'styled-components'
+import {ToccoTheme} from 'tocco-theme'
+import widgetTheme from "tocco-theme/src/ToccoTheme/widgetTheme"
```

Comment:
Hm we should not have imports via `src` folder, we should only have top level imports.
That's why I would suggest that you create a new Folder `WidgetTheme` next to `ToccoTheme` and add the widget theme there (and add a default export for it. So that you can import it like
```
import {ToccoTheme, WidgetTheme} from 'tocco-theme'
```